packages/typespec-test/test/authoring/generated/typespec-ts/src/authoringClient.ts

```diff
@@ -18,7 +18,7 @@ export default function createClient(
   options: ClientOptions = {},
 ): AuthoringClient {
   const baseUrl = options.baseUrl ?? `${endpoint}/language`;
-  options.apiVersion = options.apiVersion ?? "202ß2-05-15-preview";
+
```

Comment:
This is because default apiVerison in @service is not recognized as valid default so all query param apiVersions would be operation level param now.

I think we need to rethink this design: https://github.com/Azure/autorest.typescript/issues/2337

Reply:
I think we should change their typespec to define the default api version in a new way. instead of changing the generated code here.